src/TodoList.js

```diff
@@ -1,17 +1,28 @@
 import React from "react";
+import TodoListItem from "./TodoListItem";
 
-let todoList = [
-  { id: 1, title: "Homework" },
-  { id: 2, title: "Exercise" },
-  { id: 3, title: "Study" },
-];
+// let todoList = [
+//   { id: 1, title: "Homework" },
+//   { id: 2, title: "Exercise" },
+//   { id: 3, title: "Study" },
+// ];
 
-let list = todoList.map((item) => <li key={item.id}>{item.title}</li>);
+// let list = todoList.map((item) => <li key={item.id}>{item.title}</li>);
 
 function TodoList() {
+  let todoList = [
+    { id: 1, title: "Homework" },
+    { id: 2, title: "Exercise" },
+    { id: 3, title: "Study" },
+  ];
+
   return (
     <div>
-      <ul>{list}</ul>
+      <ul>
+        {todoList.map((item) => (
+          <TodoListItem key={item.id} todoList={todoList} item={item} />
+        ))}
+      </ul>
     </div>
```

Comment:
Good job here too! 🎖️ 

Watch out here, you don't need to pass the whole list to each item. What do you think you need to change here?

Reply:
delete todoList={todoList} ?

Reply:
Yup yup!! 💯